### DISCUSSION TITLE
How do you run BTCPay locally not for development of the server?

### DISCUSSION THREAD

**Discussion by sreyas-voruganti** [btcpayserver/btcpayserver#3147] [Technical Support]
I'm developing a webapp that requires BTCPay, so I want to run it in testnet/regtest mode locally. However I don't want to manually build and get all the dependencies since I'm not developing the BTCPay server as shown in the docs. Is there a way to use docker to run BTCPay in testnet/regtest mode using docker without nginx (on a localhost:port url)?

**Comment by woutersamaey** [comment]
The easiest way would be to run it with docker using this: https://github.com/btcpayserver/btcpayserver-docker
